<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>PhyloToAST by smdabdoub</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/github-light.css">
    <meta name="viewport" content="width=device-width">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <h1>PhyloToAST</h1>
        <p>Tools for phylogenetic data analysis including visualization and cluster-computing support.</p>

        <p class="view"><a href="https://github.com/smdabdoub/phylotoast">View the Project on GitHub <small>smdabdoub/phylotoast</small></a></p>


        <ul>
          <li><a href="https://github.com/smdabdoub/phylotoast/zipball/master">Download <strong>ZIP File</strong></a></li>
          <li><a href="https://github.com/smdabdoub/phylotoast/tarball/master">Download <strong>TAR Ball</strong></a></li>
          <li><a href="https://github.com/smdabdoub/phylotoast">View On <strong>GitHub</strong></a></li>
        </ul>
      </header>
      <section>
        <h1>
<a id="phylotoast" class="anchor" href="#phylotoast" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>PhyloToAST</h1>

<p><a href="https://pypi.python.org/pypi/phylotoast"><img src="https://img.shields.io/pypi/v/phylotoast.svg?style=plastic" alt="Latest version on PyPI"></a>
<a href="https://pypi.python.org/pypi/phylotoast"><img src="https://img.shields.io/pypi/l/phylotoast.svg?style=plastic" alt="License"></a>
<a href="https://pypi.python.org/pypi/phylotoast"><img src="https://img.shields.io/pypi/format/phylotoast.svg?style=plastic" alt="Download format"></a>
<a href="https://travis-ci.org/smdabdoub/phylotoast"><img src="https://img.shields.io/travis/smdabdoub/phylotoast.svg?style=plastic" alt="Travis CI build status"></a>
<a href="https://bioconda.github.io/recipes/phylotoast/README.html"><img src="https://img.shields.io/badge/install%20with-bioconda-4682b4.svg?style=plastic" alt="PhyloToAST on bioconda"></a></p>

<p><img src="https://github.com/smdabdoub/phylotoast/raw/master/images/readmepic.png" alt=""></p>

<p>The PhyloToAST project is a collection of python code and scripts that
modify the QIIME [1] pipeline by adding/changing several
steps including: support for cluster-computing, multiple primer support
(eliminate primer bias) [2], enhanced support for species-specific
analysis, and additional visualization tools.</p>

<h2>
<a id="installation" class="anchor" href="#installation" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Installation</h2>

<p>To install PhyloToAST from PyPI:</p>

<pre><code>$ pip install phylotoast
</code></pre>

<p>From source:</p>

<pre><code>$ python setup.py install
</code></pre>

<h2>
<a id="documentation" class="anchor" href="#documentation" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Documentation</h2>

<p>Full documentation for the scripts and code is available at the
<a href="http://phylotoast.readthedocs.org">PhyloToAST documentation site</a></p>

<h2>
<a id="requirements" class="anchor" href="#requirements" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Requirements</h2>

<p>The list of required modules will vary depending on which executable scripts and/or
parts of the API you may use. For this reason there are no required dependencies
that will be automatically installed along with PhyloToAST. Each executable script will
check that the required libraries are installed and will print a message if any are not
found. </p>

<p>If you would like to install everything up front, the following is a complete list of libraries
that are used in PhyloToAST:</p>

<ul>
<li>numpy</li>
<li>scipy</li>
<li>matplotlib &gt;= 1.5.0</li>
<li>biopython &gt;= 1.60 </li>
<li>scikit-bio</li>
<li>scikit-learn</li>
<li>pandas</li>
<li>statsmodels</li>
<li>brewer2mpl</li>
<li>biom-format &gt;= 2.1.5</li>
<li>h5py (for parsing BIOM v2.x format files)</li>
</ul>

<h2>
<a id="source" class="anchor" href="#source" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Source</h2>

<p>The <a href="http://github.com/smdabdoub/phylotoast">PhyloToAST source</a> is hosted on github.</p>

<h2>
<a id="citing" class="anchor" href="#citing" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Citing</h2>

<p>Dabdoub, S.M. et al., <em>PhyloToAST: Bioinformatics tools for
species-level analysis and visualization of complex microbial communities</em>.
2016. (Manuscript in revision).</p>

<h2>
<a id="publications-using-phylotoast" class="anchor" href="#publications-using-phylotoast" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Publications using PhyloToAST</h2>

<p>Tsigarida and Dabdoub et al., <em>The Influence of Smoking on the Peri-Implant
Microbiome</em>. Journal of Dental Research, 2015; 
<a href="http://dx.doi.org/10.1177/0022034515590581">doi: 10.1177/0022034515590581</a></p>

<p>Mason et al., <em>The subgingival microbiome of clinically healthy current
and never smokers</em>. The ISME Journal, 2014; 
<a href="http://dx.doi.org/10.1038/ismej.2014.114">doi:10.1038/ismej.2014.114</a></p>

<p>Dabdoub et al., <em>Patient-specific Analysis of Periodontal and Peri-implant Microbiomes</em>.
Journal of Dental Research, 2013; <a href="http://dx.doi.org/10.1177/0022034513504950">doi: 10.1177/0022034513504950</a></p>

<h2>
<a id="references" class="anchor" href="#references" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>References</h2>

<p>[1] J Gregory Caporaso, et al., <em>QIIME allows analysis of
high-throughput community sequencing data</em>. Nature Methods, 2010;
<a href="http://dx.doi.org/10.1038/nmeth.f.303">doi:10.1038/nmeth.f.303</a></p>

<p>[2] Kumar PS, et al., <em>Target Region Selection Is a Critical Determinant
of Community Fingerprints Generated by 16S Pyrosequencing</em>. PLoS ONE
(2011) 6(6): e20956. <a href="http://dx.doi.org/10.1371/journal.pone.0020956">doi:10.1371/journal.pone.0020956</a></p>
      </section>
      <footer>
        <p>This project is maintained by <a href="https://github.com/smdabdoub">smdabdoub</a></p>
        <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></small></p>
      </footer>
    </div>
    <script src="javascripts/scale.fix.js"></script>
    
  </body>
</html>
